RoddyCore/test/de/dkfz/roddy/tools/RoddyIOHelperMethodsTest.groovy

```diff
@@ -179,4 +184,30 @@ public class RoddyIOHelperMethodsTest {
                 assert res == RoddyIOHelperMethods.convertUMaskToAccessRights(rights);
         }
     }
+
+    @Test
+    public void testSplitPathname() throws Exception {
+        assert RoddyIOHelperMethods.splitPathname("") == new ArrayList()
+        assert RoddyIOHelperMethods.splitPathname("/") == new ArrayList()
+        assert RoddyIOHelperMethods.splitPathname("/a/b") == ["a", "b"]
+        assert RoddyIOHelperMethods.splitPathname("//a/b/") == ["a", "b"]
+    }
+
+    @Test
+    public void testMatchComponentInPath() throws Exception {
```

Comment:
Would findComponentIndex be better than matchComponent? Actually you retrieve an index.